iris_grib/__init__.py

```diff
@@ -1028,19 +1028,6 @@ def save_grib2(cube, target, append=False, **kwargs):
     save_messages(messages, target, append=append)
 
 
-def as_pairs(cube):
-    """
-    .. deprecated:: 1.10
```

Comment:
My take on it is that it's deprecated in `iris` as a fall-back implementation. `iris-grib` on the otherhand doesn't need to honour that deprecation. It's a brand new world, so purge the deprecations as per #8